tests/dhcp_server/test_dhcp_server_multi_vlans.py

```diff
@@ -250,6 +237,7 @@ def test_single_ip_assignment(
             server_id=gateway,
             net_mask=net_mask
         )
+    clean_dhcp_server_config(duthost)
```

Comment:
Would it be better to clean by fixcture?

Reply:
Yes, I moved the common config clean to new fixture clean_dhcp_server_after_test